### PR TITLE
Enable TSA bugfiling

### PR DESCRIPTION
**Pull request by heyams**
[TSA bug filing config](https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/4923/SDL-Bugfiling-(TSA)?anchor=tsa-bug-filing-configuration)

Fix [Compliance 9744465](https://msazure.visualstudio.com/One/_sprints/taskboard/TelReachSDKTeam/One/AEP/Cobalt/2105?workitem=9744465)

### CODE REVIEW

File: .config/tsaoptions.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "instanceUrl"        :  "https://msazure.visualstudio.com",
+  "areaPath"           :  "One\\TelReachSDKTeam",
+  "projectName"        :  "One",
+  "iterationPath"      :  "One",
+  "notificationAliases":  [
+    "Trask.Stalnaker@microsoft.com",
```
Comment (heyams) [inline .config/tsaoptions.json:7]: @trask can i use appinsightsjavahelp@microsoft.com or create a separate dl to exclude Julia? She probably doesn't need to know about compliance alerts.

File: .config/tsaoptions.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "Helen.Yang@microsoft.com"
+  ],
+  "codebaseAdmins"     : [
+    "Trask.Stalnaker@microsoft.com",
```
Comment (heyams) [inline .config/tsaoptions.json:11]: same here.

File: .config/tsaoptions.json (hunk 1 above)
Comment (trask) [reply .config/tsaoptions.json:7]: yes, appinsightsjavahelp@microsoft.com sounds good 👍

File: .config/tsaoptions.json (hunk 2 above)
Comment (trask) [reply .config/tsaoptions.json:11]: the doc for codebaseAdmins doesn't appear to be email addresses? https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/4923/SDL-Bugfiling-(TSA)?anchor=codebase-admins

Comment (heyams) [reply .config/tsaoptions.json]: removed.. realized that i don't need to change it.. whatever the default will be fine.

**Review by trask — APPROVED** [review]

**Pull request merged**